src/services/item/plugins/embeddedLink/utils.ts

```diff
@@ -1,7 +1,7 @@
-import isUrlHttp from 'is-url-http';
+import urlHttp from 'url-http';
 
-export const isValidUrl = (url: string) => {
-  return isUrlHttp(ensureProtocol(url));
+export const isValidUrl = (url: string): boolean => {
+  return !!urlHttp(ensureProtocol(url));
```

Comment:
Should we use `Boolean` instead of double negation here?

```suggestion
  return Boolean(urlHttp(ensureProtocol(url)));
```